projects/uitpas/reference/uitpas.json

```diff
@@ -4352,6 +4352,9 @@
           },
           "403": {
             "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "Not Found. Error type:\n\nhttps://api.publiq.be/probs/url/not-found"
```

Comment:
In which case would this endpoint return a 404? If there are no results found I'd expect it to return 200 but with an empty list.

Reply:
this can be the case when postalCode or city does not validate